modules/ROOT/partials/misc/ui-languages.adoc

```diff
@@ -41,9 +44,13 @@
 | Thai (Thailand)                                                            | th_TH | `./langs/th_TH.js` |
 | Turkish                                                                    |  tr   |  `./langs/tr.js`   |
 | Ukrainian                                                                  |  uk   |  `./langs/uk.js`   |
-{% else %}
+|===
+endif::[]
+ifeval::[{forModuleLoaders} == false]
```

Comment:
This doesn't have the same meaning, as if `forModuleLoaders` is not defined then this won't render whereas it would have previously. As such, this should use `!= true` instead.
```suggestion
ifeval::[{forModuleLoaders} != true]
```

Reply:
done